scripts/release.js

```diff
@@ -35,6 +31,12 @@ const getConfig = ({ gitBranchName }) => {
 		githubConfig.failComment = false;
 	}
 
+	// Only publish alpha and release branches to NPM.
+	shouldPublishOnNPM = shouldPublishOnNPM && ["alpha", "release"].includes(branchType);
```

Comment:
nit: `shouldPublishOnNPM` doesn't need to be defined above. It can be defined here like:

```suggestion
const shouldPublishOnNPM = Boolean( process.env.NPM_TOKEN ) && ["alpha", "release"].includes(branchType);
```

Reply:
Good call, updated in 1b5b2a1!